project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.11
+sbt.version=1.2.8
```

Comment:
I haven't gone to 1.3 because they broke sbt-dependency-graph and didn't implement the `coursierDependencyTree` command, so there's literally no way to get SBT 1.3 to show a dependency tree... 🤕 

https://github.com/sbt/sbt/issues/4707

Reply:
Fix in SBT 1.3.13. I've updated